kong.conf.default

```diff
@@ -173,17 +173,56 @@
                                  #   It runs DB-less and receives configuration
                                  #   updates from a control plane node.
 
+#cluster_mtls = shared           # Sets the verification between nodes of the
+                                 # cluster.
+                                 #
+                                 # Valid values to this setting are:
+                                 #
+                                 # - `shared`: use a shared certificate/key
+                                 #   pair specified with the `cluster_cert`
+                                 #   and `cluster_cert_key` settings.
+                                 #   Note that CP and DP nodes have to present
+                                 #   the same certificate to establish mTLS
+                                 #   connections.
+                                 # - `pki`: use `cluster_ca_cert` and
```

Comment:
Recommendation: "`pki`: use `cluster_ca_cert`, `cluster_server_name` and `cluster_cert` for verification,"  should be more accurate. That way the user knows exactly what needs to be set in the `pki` mode.